docs/book/src/developer/providers/implementers-guide/create_api.md

```diff
@@ -62,3 +62,68 @@ As the deleted comments request, run `make manager manifests` to regenerate some
 git add .
 git commit -m "Added cluster types"
 ```
+
+# Registering APIs in the scheme
+
+To enable clients to encode and decode your API, your types must be able to be registered within a [scheme].
+
+[scheme]: https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#Scheme
+
+By default, Kubebuilder will provide you with a scheme builder like:
```

Comment:
Do we have an issue in Kubebuilder to try steer away from the current pattern?

Reply:
Not yet, I'll make one